src/types/vendors.ts

```diff
@@ -53,3 +53,101 @@ export type UpdateVendorPayload = {
   tags?: string[]
   comment?: string
 }
+
+export type SkillsData = {
+  id: string
+  name: string
+}
+
+export type GetSkillsPayload = {
+  data?: SkillsData[]
+}
+
+export interface Prices {
+  id?: string
+  vendor_id?: string
+  skill_id?: string
+  src_lang_classifier_value_id?: string
+  dst_lang_classifier_value_id?: string
+  character_fee?: number
+  word_fee?: number
+  page_fee?: number
+  minute_fee?: number
+  hour_fee?: number
+  minimal_fee?: number
+}
+export interface UpdatedPricesData {
+  id?: string
+  character_fee?: number
+  word_fee?: number
+  page_fee?: number
+  minute_fee?: number
+  hour_fee?: number
+  minimal_fee?: number
+}
+
+export type CreatePricesPayload = {
+  data?: Prices[]
+}
+
+export type UpdatePricesPayload = {
+  data?: UpdatedPricesData[]
+}
+export type DeletePricesPayload = {
+  id?: string[]
+}
+
+export type PricesData = {
```

Comment:
**[blocker]**

Hmm, this looks a lot like another copy of the "Prices" type here and the "Price" type in "types/price".

These should be unified. If the difference is that from some endpoint we get less fields and from other endpoints we get more fields then we should have 1 type defined with the fields we always get and 1 extra type that extends the first one, with the new fields added.